.github/integration_tests/nonexistent_repo.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Integration test to ensure CI fails when a non-existent repository is added
+
+# set git credentials
+git config user.name dfinity
+git config user.email dfinity@dfinity.org
+
+gh auth login
+
+# create new branch
+git checkout -b integration-test-1
```

Comment:
That will fail when several workflows (from different PRs) will run in parallel.

Reply:
It will only run if someone makes an update to the workflows or the code - it won't run if someone updates `open-repositories.txt`

Reply:
I think it will currently also run on every merge queue

Reply:
Oh good, point, I'll remove the merge group.